storage/mysql/README.md

```diff
@@ -27,3 +27,26 @@ If nothing is found for the key then <i>null</i> is returned.
 
 The reconnect retry timeout in milliseconds can be changed by adding a line to <b>settings.js</b>
     <pre>mysqlReconnectTime: 30000,</pre></p>
+
+
+Preparing Queries
+-----
+```javascript
+msg.payload=[24, 'Helloworld'];
+msg.topic="INSERT INTO database (`id`, `field1`) VALUES (?, ?);"
+return msg;
+```
+
+with named parameters:
+
+```javascript
+msg.payload={}
+msg.payload.id=42;
```

Comment:
Example may be clearer if you used myid (or something other than id) instead so you could see which were the mysql variables and which were the javascript ones